gradle-plugin/src/main/kotlin/arrow/meta/plugin/gradle/ArrowGradlePlugin.kt

```diff
@@ -20,10 +20,5 @@ class ArrowGradlePlugin : Plugin<Project> {
 
   override fun apply(project: Project): Unit {
     project.extensions.create("arrow", ArrowExtension::class.java)
```

Comment:
You should not delete that. Otherways we can not add the compiler-plugin to the repo. Once we deploy the compiler plugin we can change that to add the dependency. But as for now, we would keep it.

Reply:
Because of that to use plugin you must clone arrow-meta in specific directory. Gradle plugin has mechanism of automatically downloading and applying kotlin compiler plugin from maven repository.

Reply:
Can you explain how that works. We did not have that experience that Gradle does that?

Reply:
@jereksel Thanks for the explanation in Slack. The problem was that we were pointing to the gradle-plugin itself, which lead to the configurations we had back then to add the compiler-plugin manually.